recipes-openxt/xctools/dom0-rpc-broker-config.bb

```diff
@@ -0,0 +1,16 @@
+DESCRIPTION = "Dom0 policy file for rpc-broker"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+SRC_URI = " \
+    file://rpc-broker-dom0.rules\
+"
+
```

Comment:
You can `inherit allarch` since this config file doesn't depend on the architecture.  Naming it rpc-broker-config-dom0 may nicer since then the two recipes will be together in the listing.

Reply:
Will do.